packages/scene-composer/src/components/panels/SceneNodeInspectorPanel.tsx

```diff
@@ -141,7 +142,7 @@ export const SceneNodeInspectorPanel: React.FC = () => {
 
         {/* If the component is a Tag and there is no overlay component in the selected node, then an empty
           overlay sections is added. */}
-        {component.type === KnownComponentType.Tag && !isOverlayComponent && (
+        {overlayEnabled && component.type === KnownComponentType.Tag && !isOverlayComponent && (
```

Comment:
Nit-pick: We've been getting comments on readability. You could update this to break out the logic a little more to make it easier for readers. Ex:
`const isTag = component.type === KnownCOmponentType.Tag`
`(overlayEnabled && isTag && !isOverlayComponent && (...`

Reply:
sure. I will update it in next PR for some updates to the visibility toggle